app/components/article-content.js

```diff
@@ -44,9 +44,6 @@ export default Component.extend(
     lang: reads('wikiVariables.language.content'),
     dir: reads('wikiVariables.language.contentDir'),
     isFastBoot: reads('fastboot.isFastBoot'),
-    affiliateUnit: computed('displayTitle', function () {
```

Comment:
`  2:10  error  'computed' is defined but never used  no-unused-vars`

Reply:
5b1746d